dev/com.ibm.websphere.org.eclipse.microprofile.reactive.streams.operators.1.0/bnd.bnd

```diff
@@ -18,12 +16,9 @@ javac.source: 1.8
 javac.target: 1.8
 
 Bundle-SymbolicName: com.ibm.websphere.org.eclipse.microprofile.reactive.streams.operators.1.0; singleton:=true
-Bundle-Name: MicroProfile Reactive Streams Operators Bundle
```

Comment:
There are 570 occurrences of this attribute in WAS bnd files including lots from our CDI projects.